framework/infrastructure/traffic_director.py

```diff
@@ -693,18 +735,45 @@ def create_forwarding_rule(self, src_port: int):
             name, src_port, self.target_proxy, self.network_url
         )
         self.forwarding_rule = resource
+
+        return resource
+
+    def create_forwarding_rule_ipv6(self, src_port: int):
+        name = self.make_resource_name(self.FORWARDING_RULE_NAME_IPV6)
+        src_port = int(src_port)
```

Comment:
[Minor] Do we still need converting `src_port` here if it was passed in as `int` already?